app/furniture/marketplace.rb

```diff
@@ -1,15 +1,5 @@
 # @see features/furniture/marketplace.feature.md
 class Marketplace
-  def self.append_routes(router)
-    router.resources :marketplaces, only: [:show, :edit, :update], module: "marketplace" do
-      router.resources :products
-      router.resources :carts do
-        router.resources :cart_products
-      end
-      router.resources :checkouts, only: [:show, :create]
-    end
-  end
```

Comment:
I moved these to `furniture/marketplace/routes.rb` which I think will make it easier to find routes because you can use a fuzzy-finder and type `marketplace routes` and get the routes; instead of having to remember that they live in `furniture/marketplace.rb`